src/app/buy-deso/buy-deso-megaswap/buy-deso-megaswap.component.html

```diff
@@ -11,6 +11,7 @@
   </ng-template>
   <div class="d-flex justify-content-center">
     <button
+      data-control-name="buy-deso-finish-flow-button"
```

Comment:
I think we've got a public key at this point - should we log the public key of the user who finished the flow?

Reply:
yep i added a thing to just always pull it from the query params. if it's not there yet it'll just be empty which seems fine. The whole stream of interactions should be back associated once diamond identifies the user after login.